force-app/main/default/classes/DMLOperations.cls

```diff
@@ -198,10 +274,15 @@ public with sharing class DMLOperations {
      */
     public static void insertAndDeleteLeads(List<String> leadNames) {
         // Loop through each Lead name
-
+        List <Lead> insertLeads = new List<Lead>();
```

Comment:
Watch your spacing. Also, we tend to start method names with verbs (`insertLeads`). Try to name objects using nouns -- plural for lists and sets, singular for a single object. Here, your list name would be written as `leadsToInsert`. 
```suggestion
        List<Lead> leadsToInsert = new List<Lead>();
```